src/main/java/core/basesyntax/utils/impl/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.utils.impl;
+
+import core.basesyntax.model.ResultData;
+import core.basesyntax.utils.ReportGenerator;
+import java.util.List;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+
+    @Override
+    public String getReport(List<ResultData> resultDataList) {
+        StringBuilder stringBuilder = new StringBuilder("fruit,quantity\n");
```

Comment:
Make constants for the header and separator. Also better use System.lineseparator()